### PR TITLE
[WIP] add resource quotas

### PR DESCRIPTION
**Pull request by JulianKahnert**

### CODE REVIEW

File: project-template/values.yaml [hunk 1]
```diff
@@ -105,7 +105,7 @@ servePublicly: true
 #  - dev
 
 # mysqlBackup:
-#   schedule: "*/1 * * * *"
+#   schedule: "30 4 * * *"
 #   # for mysql 5.7 this might be changed according to: https://github.com/worldiety/mysql-scp-backup
```
Comment (niveau0) [inline project-template/values.yaml:110]: I would make this an extra commit

File: user-creation/templates/user-manifests.yaml [hunk 2]
```diff
@@ -36,6 +36,20 @@ spec:
       memory: 32Mi
     type: Container
 
+---
+# Resource Qutas per user, e.g. namespace
```
Comment (niveau0) [inline user-creation/templates/user-manifests.yaml]: typo :) "Quotas"

File: user-creation/values.yaml [hunk 3]
```diff
@@ -2,4 +2,8 @@
 
 # projects:
 #   - name: example-project-1
-#   - name: example-project-2
\ No newline at end of file
+#     cpuLimit: 1000m
```
Comment (niveau0) [inline user-creation/values.yaml:16]: maybe add some information or link how to interpret these values, e.g.
1000m means? one full CPU, a quarter of 4 CPUs or simply if abstracted, 1000 CPU milliseconds (how to interpret that?)
https://medium.com/@betz.mark/understanding-resource-limits-in-kubernetes-cpu-time-9eff74d3161b

**Review by niveau0 — CHANGES_REQUESTED** [review]
added some inline comments

File: project-template/values.yaml (hunk 1 above)
Comment (JulianKahnert) [reply project-template/values.yaml:110]: this was an unsaved changed from #34 . I just wanted to have a reasonable value in `values.yaml`

File: user-creation/templates/user-manifests.yaml [hunk 4]
```diff
@@ -29,13 +29,30 @@ metadata:
...
+      cpu: 125m
       memory: 512Mi
     defaultRequest:
       cpu: 10m
       memory: 32Mi
     type: Container
 
+---
+# Resource Quotas per user, e.g. namespace
+# https://kubernetes.io/docs/tasks/administer-cluster/quota-api-object/
```
Comment (JulianKahnert) [inline user-creation/templates/user-manifests.yaml:44]: documentation: request can never be greater than the limit => a malicous user can not set the request to e.g. 15GB while conforming to a set limit of 2GB

**Pull request merged**